.eslintrc

```diff
@@ -1,6 +1,9 @@
 {
   "env": { "es6": true, "node": true },
   "extends": "rapid7/node",
+  "parserOptions": {
+    "ecmaVersion": 2018
```

Comment:
breaking compatibility for no good reason?

Reply:
check the commit comments
https://github.com/rapid7/r7insight_node/pull/21/commits/60d54ed15d6d4f90f9ab980c4c14c4fc03d67204

Reply:
without it we can't use spread